data/Warm_Cold.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+number = random.randint(1, 50)
+value = input("Please enter a number:\n")
+counter = 0
+while True:
+    counter += 1
+    if number == int(value):
```

Comment:
Tworzenie int() z value w tym miejscu i tych poniżej nie jest dobrym sposobem bo muszisz zmieniać ten typ aż 2 razy lepiej zrobić to raz na samym początku.

Reply:
Zmienione. Zmiana jest wczytywana od razu jako int